docs/components/integration-framework/connectors/out-of-the-box-connectors/googledrive.md

```diff
@@ -0,0 +1,245 @@
+
+To use a **Google Drive Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu or create a new connector task by using the **Append Connector** context menu. Follow our [guide on using Connectors](../use-connectors.md) to learn more.
+
+## Make your Google Drive Connector executable
+
+To make the **Google Drive Connector** executable, fill out the mandatory fields highlighted in red in the properties panel.
+
+### Create a new folder
+
+![Google Drive Connector new folder example](../img/connectors-googledrive-new-folder-filled.png)
+
+1. Set required credentials in the **Authentication** section. See the [relevant appendix entry](#how-can-i-authenticate-my-connector) to find out more.
+2. In the **Select Operation** section, set the field value **Operation Type** as **Create Folder**.
+3. In the **Operation Details** section, set the field **Folder name** as the desired name of a folder you wish to create. For example, `MyNewFolder`. Alternatively, you could use a FEEL expression.
+4. _(optional)_ In the **Operation Details** section, set the field **Parent folder ID** to the desired parent, inside which a new folder will be created. Keep in mind that if not specified, a new folder will be created in the Google Drive root folder of a user who owns the OAuth token.
+5. _(optional)_ In the **Operation Details** section, you can set the **Additional properties or metadata** field to Google Drive compatible properties. For example, _description_ of the folder. This property requires FEEL input. Check [the appendix](#what-are-the-limitations-of-the-additional-properties-or-metadata) for known values and limitations.
+
+### Create a new file from a template
+
+![Google Drive Connector new file from template example](../img/connectors-googledrive-new-file-filled.png)
+
+1. Set required credentials in the **Authentication** section. See the [relevant appendix entry](#how-can-i-authenticate-my-connector) to find out more.
+2. In the **Select Operation** section, set the field value **Operation Type** as **Create File from template**.
+3. In the **Operation Details** section, set the field **File name** as the desired name of a folder you wish to create. You can use FEEL expressions here.
+4. In the **Operation Details** section, set the field **Template ID** of the desired template.
+5. _(optional)_ In the **Operation Details** section, set the field **Parent folder ID** to the desired parent, inside which a new file will be created. Keep in mind that if not specified, a new folder will be created in the Google Drive root folder of a user who owns the OAuth token.
+6. In the **Operation Details** section, set the field **Template variables** as desired variables that will be applied to the template. The template variables a compatible with the Google Docs Requests API. This property requires FEEL input.
```

Comment:
Pls update text from: "The template variables a compatible with the Google Docs Requests API."
to: "The template variables are compatible with the Google Docs Requests API."

Reply:
Addressed in a new revision. Will publish shortly.